### PR TITLE
Chore improve README.md file

### PR DESCRIPTION
**Pull request by getspooky**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,49 @@
...
-  <summary>Boom</summary>
-  <p>
+## Getting Started
+
+Visit [https://alephjs.org/docs/get-started](https://alephjs.org/docs/get-started) to get started with Aleph.js.
+
+
+## Description
+
+Aleph.js (or **Aleph** or **א** or **阿莱夫**, **ˈɑːlɛf**) is a fullstack framework in [Deno](https://deno.land/), inspired by [Aleph.js](https://alephjs.org/). The name is taken from the book **The Aleph** by [Jorge Luis Borges](http://phinnweb.org/links/literature/borges/aleph.html).
```
Comment (ije) [inline README.md:15]: inspired by Next.js

File: README.md [hunk 2]
```diff
@@ -1,26 +1,49 @@
...
+
+Different with Aleph.js, Aleph.js doesn't need webpack or other bundler since it uses the [ES Moudule](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules) syntax during development. Every module only needs to be compiled once, and then cached on the disk. When a module changes, Aleph.js just needs to re-compile that single module. There is no time wasted re-bundling everytime a change is made. This, along with Hot Module Replacement (HMR) and Fast Refresh, leads to instant updates in the browser.
```
Comment (ije) [inline README.md:17]: Different with Next.js

File: README.md (hunk 1 above)
Comment (getspooky) [reply README.md]: @ije fixed

File: README.md (hunk 2 above)
Comment (getspooky) [reply README.md]: fixed

**Comment by ije** [comment]
lgtm, thanks 🙏

**Pull request merged**